web/src/containers/loans/Loan.js

```diff
@@ -28,101 +28,53 @@ import walletService from "../../services/wallet.service";
 import LoadingPageAnimation from "../../components/LoadingPageAnimation";
 import documentsService from "../../services/documents.service";
 
-const LoanActive = ({ loan }) => {
-  const [documents, setDocuments] = useState([]);
-  useEffect(() => {
-    const fetchDocuments = async () => {
-      const docs = await documentsService.getLoanPublicDocuments(loan.id);
-      setDocuments(docs);
-    };
-    fetchDocuments();
-  }, []);
+const fetchInvestments = async (loan, setInvestment) => {
+  const account = walletService.checkAccount();
+  if (loan && account != null) {
+    const investment = await InvestmentsService.getInvestmentInLoan(
+      loan.id,
+      account
+    );
+    console.log("investment:", investment);
+    setInvestment(investment.nrTokens);
+  }
+};
```

Comment:
The pattern used is to define these type of functions inside the useEffect. This way you don't have to pass functions outside of the component